classes/WP_CLI/Action/Delete.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * WP-CLI command: action-scheduler action delete
+ */
+class ActionScheduler_WPCLI_Action_Delete_Command extends ActionScheduler_WPCLI_Command {
```

Comment:
do you think if we should support a bulk delete option as well, for example, deleting by hook name? folks may want to use this command to control number of action, in which case deleting by hook would be more useful than deleting by id.

Reply:
Such a task should already be doable with something like this:

```shell
wp action-scheduler action delete $( wp action-scheduler list --hook=action_hook --format=ids )
```